src/listener/Heading.php

```diff
@@ -31,6 +31,11 @@ public function process(Line $line)
     public function render(\nadar\quill\Lexer $lexer)
     {
         foreach ($this->picks() as $pick) {
+            if (in_array($pick->heading, [1, 2, 3, 4, 5, 6], $strict=true) === false) {
```

Comment:
This could be handled as separate PR.

Reply:
I understand, I'm also a big fan of separate PRs!

I chose for this as it is also needed to prevent html injection. If we don't throw an exception we should escape the value.

I can also setup a separate PR, but as long as that is not merged, here the value should be escaped.

Reply:
yes, i agree its part of html injection prevention.